packages/typescript/package.json

```diff
@@ -58,7 +58,8 @@
     "@rollup/plugin-commonjs": "^11.1.0",
     "@rollup/plugin-typescript": "^5.0.2",
     "buble": "^0.20.0",
-    "rollup": "^2.14.0"
+    "rollup": "^2.14.0",
+    "typescript": "^4.0.0"
```

Comment:
Let's update to the latest version (4.1.2)

Reply:
done (though the `^` prefix means get the latest 4.x version in either case, so they are equivalent)